.github/tools/install-mpi.sh

```diff
@@ -83,11 +83,6 @@ case "$os" in
           echo "Not taking any action."
           exit 0
         fi
-        if [ -n "${I_MPI_ROOT}" ]; then
-          echo "MPI is already installed at I_MPI_ROOT=${I_MPI_ROOT}."
-          echo "Not taking any action."
-          exit 0
-        fi
```

Comment:
You should not need to remove this.

Reply:
Okay, I can't remember why I removed this. I'll put it back and see what happens.

Reply:
OK makes sense then. But then why did you add line 161 too, if we are to ignore I_MPI_ROOT?